workflows/models.py

```diff
@@ -66,7 +67,7 @@ def get_reference_genome(self):
         reference_genome = self.analysis_info['reference_genome']['reference_genome']
         if reference_genome not in self.reference_genome_choices:
             raise Exception('Unrecognized reference genome {}'.format(reference_genome))
-        return self.reference_genome_choices[reference_genome]
+        return reference_genome
```

Comment:
why this?

Reply:
When I ran a pipeline, I was getting keyerrors because it was trying to find an identifier hg19 rather than grch37